common/utils/file.go

```diff
@@ -17,8 +17,6 @@ func ReadSecretFile(filePath string) (pulumi.StringPtrOutput, error) {
 	return s, nil
 }
 
-func WriteStringCommand(content pulumi.StringInput, filePath string) pulumi.StringInput {
-	return pulumi.Sprintf(`cat <<EOF > %s
-%s
-EOF`, filePath, content)
+func WriteStringCommand(filePath string) pulumi.StringInput {
+	return pulumi.Sprintf(`cat - > %s`, filePath)
```

Comment:
<!-- Praises highlight something positive. Try to leave at least one of these comments per review. Do look for something to sincerely praise. -->
**👏 praise**
TIL ! Wow did not know we could open a file as a stream, super cool.